flask_restless/helpers.py

```diff
@@ -556,16 +583,18 @@ def strings_to_dates(model, dictionary):
     """
     result = {}
     for fieldname, value in dictionary.items():
-        if is_date_field(model, fieldname) and value is not None:
+        # Interval fields must be checked before date fields because Interval
+        # fields impl DateTimes.
+        if (is_interval_field(model, fieldname) and value is not None and
+                isinstance(value, int)):
```

Comment:
This line should be `and isinstance(value, int)):`, and it should be aligned with the first character after the opening parenthesis on the previous line.